build-javascript/action.yml

```diff
@@ -22,21 +22,21 @@ runs:
   using: 'composite'
   steps:
     - name: Build package
-      uses: jahia/jahia-modules-action/build-step-npm@v2
+      uses: jahia/jahia-modules-action/build-step-javascript@chore/rename-npm-to-javascript
```

Comment:
Except if you need to perform particular tests with the branch, I'd recommend using `@v2` in the PR (not to forget to update it afterwards)